src/pixbuf.c

```diff
@@ -40,6 +30,20 @@ void pixbuf_save_state_to_folder(GdkPixbuf *pixbuf, char *folder) {
   g_free(message);
 }
 
+void pixbuf_save_state_to_folder(GdkPixbuf *pixbuf, char *folder) {
+
```

Comment:
Can you apply a quick `clang-format` to your patch? I might have not configured github actions to run on pull requests.

Reply:
There is `.clang-format` file in the root: https://github.com/jtheoof/swappy/blob/master/.clang-format

Reply:
done